Python/1662. Check If Two String Arrays are Equivalent.py

```diff
@@ -0,0 +1,26 @@
+# 1662. Check If Two String Arrays are Equivalent
+
+# Given two string arrays word1 and word2, return true if the two arrays represent the same string, and false otherwise.
+
+# A string is represented by an array if the array elements concatenated in order forms the string.
+
+
+# Example 1:
+
+# Input: word1 = ["ab", "c"], word2 = ["a", "bc"]
+# Output: true
+# Explanation:
+# word1 represents string "ab" + "c" -> "abc"
+# word2 represents string "a" + "bc" -> "abc"
+# The strings are the same, so return true.
+# Example 2:
+
+# Input: word1 = ["a", "cb"], word2 = ["ab", "c"]
+# Output: false
+# Example 3:
+
+# Input: word1  = ["abc", "d", "defg"], word2 = ["abcddefg"]
+# Output: true
+class Solution:
+    def arrayStringsAreEqual(self, word1: List[str], word2: List[str]) -> bool:
+        return ''.join(word1) == ''.join(word2)
```

Comment:
Interesting fact I learned about `''.join()` as opposed to using concatenation is that it is more performant because it only has to allocate memory once as opposed to concatenation which will create a new place in memory for each new string and discard the old (causing a performance hit)